internal/graph/check_test.go

```diff
@@ -119,3 +121,201 @@ func TestCheckWithOneConcurrentGoroutineCausesNoDeadlock(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, resp.Allowed)
 }
+
+func TestCheckDbReads(t *testing.T) {
```

Comment:
Can we name after the field we're testing?

`TestCheckDatastoreQueryCount`

Follows along with https://github.com/openfga/openfga/pull/880/files#r1282079519